docs/enhacement/20201211-nodepool_uniteddeployment.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Proposal about nodepool and uniteddeployment
+authors:
+  - "@kadisi"
+reviewers:
+  - "@huangyuqi" 
+  - "@Fei-Guo" 
+  - "@charleszheng44"
+creation-date: 2020-12-11
+last-updated: 2020-12-11
+status: implementable
+---
+
+   * [Proposal about nodepool and uniteddeployment](#proposal-about-nodepool-and-uniteddeployment)
+      * [Glossary](#glossary)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+      * [Proposal](#proposal)
+         * [NodePool API](#nodepool-api)
+         * [UnitedDeployment API](#uniteddeployment-api)
+      * [Implementation History](#implementation-history)
+
+
+# Proposal about nodepool and uniteddeployment  
+
+## Glossary
+
+NodePool:
+    A node pool is a group of nodes within a same region in the edge computing scenario. 
```

Comment:
Maybe we'd better add this requirement for NodePool: The nodes in the NodePool are ideally accessible to each other through the Intranet.

Reply:
> Maybe we'd better add this requirement for NodePool: The nodes in the NodePool are ideally accessible to each other through the Intranet.

great